bundles/org.openhab.binding.lifx/src/main/resources/ESH-INF/thing/tile.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="lifx"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="tile">
+		<label>LIFX Tile</label>
+		<channels>
+			<channel id="color" typeId="color" />
+			<channel id="temperature" typeId="temperature" />
+			<channel id="signalstrength" typeId="system.signal-strength" />
+			<channel id="tileeffecttype" typeId="tileeffecttype" />
```

Comment:
Other lights will likely also get effects so I prefer using a channel name here that can be reused with other LIFX Things. If we name the channel `effect` we can also keep implementation details like the specific protocol being used away from end users.

```suggestion
			<channel id="effect" typeId="tileeffect" />
```